commands

```diff
@@ -26,7 +26,7 @@ case "$1" in
     fi
 
     mkdir -p "$SERVICE_ROOT" || dokku_log_fail "Unable to create service directory"
-    password=$(openssl rand -hex 16)
+    password="postgres"
```

Comment:
I was thinking that by default it would still use the random password, but that you could override it with a command line variable.  That's looking more towards getting this fork incorporated back into the original repo, which I think would be a good idea.